src/odfesqlodbc/es_parse_result.cpp

```diff
@@ -329,7 +336,7 @@ bool AssignRowData(const json_arr_it &row, const schema_type &row_schema,
 
     // Loop through and assign data
     size_t i = 0;
-    for (auto row_column = row.value_begin(); i < row_schema.size();
+    for (auto row_column = row.value_begin(); i < row_schema_size;
          ++row_column, ++i) {
         if (row_column->is_null()) {
             tuple[i].len = SQL_NULL_DATA;
```

Comment:
Line 346 seems inefficient. I'm unsure what str() returns, but this creates a temporary string and then does a copy out of it. Would be much more efficient to simply copy directly from the str() to the target tuple.

Reply:
done

Reply:
reverted since str() creates a copy